test/blocks/global-footer/test-utilities.js

```diff
@@ -0,0 +1,93 @@
+import { setViewport } from '@web/test-runner-commands';
+import { setConfig } from '../../../libs/utils/utils.js';
+import { config, loadStyles, viewports } from '../global-navigation/test-utilities.js';
+import { waitForElement } from '../../helpers/waitfor.js';
+
+export const containerSelector = '.global-footer';
+
+export const allSelectors = {
+  footerIcons: '.feds-footer-icons',
+  footerWrapper: '.feds-footer-wrapper',
+  menuContent: '.feds-menu-content',
+  menuColumn: '.feds-menu-column',
+  menuSection: '.feds-menu-section',
+  menuHeadline: '.feds-menu-headline',
+  menuItems: '.feds-menu-items',
+  featuredProducts: '.feds-featuredProducts',
+  featuredProductsLabel: '.feds-featuredProducts-label',
+  navLink: '.feds-navLink',
+  navLinkImage: '.feds-navLink-image',
+  navLinkContent: '.feds-navLink-content',
+  navLinkTitle: '.feds-navLink-title',
+  social: '.feds-social',
+  socialItem: '.feds-social-item',
+  regionPickerWrapper: '.feds-regionPicker-wrapper',
+  legalWrapper: '.feds-footer-legalWrapper',
+  privacySection: '.feds-footer-privacySection',
+  copyright: '.feds-footer-copyright',
+  privacyLink: '.feds-footer-privacyLink',
+};
+
+export const visibleSelectorsMobile = Object.fromEntries(
+  Object.entries(allSelectors).filter(([key]) => ![
+    'footerIcons',
+    'featuredProducts',
+    'featuredProductsLabel',
+    'navLink',
+    'navLinkImage',
+    'navLinkContent',
+    'navLinkTitle',
+    'menuItems',
+    'menuHeadline',
+  ].includes(key)),
+);
```

Comment:
I think the logic is a bit too convoluted. What if we went for something more like
```
const getMobileVisibleSelectors = () => {
    const { footerIcons, featuredProducts, featuredProductsLabel, navLink, navLinkImage, navLinkContent, navLinkTitle, menuItems, menuHeadline, ...mobileSelectors } = allSelectors;
    return mobileSelectors;
}
export const visibleSelectorsMobile = getMobileVisibleSelectors();
```